WORKSPACE

```diff
@@ -122,3 +122,12 @@ http_archive(
     strip_prefix = "zlib-1.2.11",
     url = "https://github.com/madler/zlib/archive/v1.2.11.tar.gz",
 )
+
+http_archive(
+    name = "range-v3",
+    patch_args = ["-p1"],
+    patches = ["//third_party/patches:range-v3.patch"],
```

Comment:
Looking at https://github.com/ericniebler/range-v3/compare/0.11.0...master I think we should just go with the latest commit on master. :P Everything after 0.11.0 appears to be bugfixes.

You can see an example of including an archive of a specifc commit if you look at stb in this file.

Reply:
I also think I have the libraries in here in alphabetical order (within their group.)